core/src/main/java/com/datastax/oss/driver/api/core/config/DriverConfigProfile.java

```diff
@@ -42,6 +43,8 @@
 
   int getInt(DriverOption option);
 
+  Map<String, String> getMap(DriverOption option);
```

Comment:
1. Could you call it `getStringMap` instead?
2. For completeness, we should have a `withStringMap` as well. It can't reuse `with(DriverOption, Object)` like the others because it's not a primitive type, but I think it should be pretty easy to adapt (basically, we need to convert the other way Map => Config).
3. nit: could you move the `getStringMap/withStringMap` pair under `withStringList`, and use the same order in the implementation class?